database/migrations/2019_10_06_173431_room_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddVerifiedToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('rom', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->integer('floor');
+            $table->integer('count_beds');
+            $table->integer('number');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('room', function (Blueprint $table) {
+            //
+        });
```

Comment:
Probably something like drop table would be perfect, but in the git history you will find plenty of examples how we implement this.